ui/desktop/tests/unit/routes/scopes/scope/authenticate/method/oidc-test.js

```diff
@@ -17,5 +20,72 @@ module(
       );
       assert.ok(route);
     });
+
+    test('setupController clears UI messages and sets authMethod on the controller', function (assert) {
+      assert.expect(2);
+
+      class FlashMessagesStub extends Service {
+        clearMessages() {
+          assert.ok(true, 'flashMessages.clearMessages was called');
+        }
+      }
+
+      this.owner.register('service:flash-messages', FlashMessagesStub);
+
+      const controller = new (class extends Controller {})();
+      const route = this.owner.lookup(
+        'route:scopes/scope/authenticate/method/oidc',
+      );
+
+      const fakeAuthMethod = { name: 'OIDC', type: 'oidc' };
+      route.modelFor = () => fakeAuthMethod;
+
+      route.setupController(controller);
+
+      assert.deepEqual(
+        controller.authMethod,
+        fakeAuthMethod,
+        'controller.authMethod was set correctly',
+      );
+    });
+
+    test('setupController clears flash messages', function (assert) {
+      assert.expect(2);
+
+      // Get the route instance
```

Comment:
nitpick: Do these comments provide extra context that the test itself does not? While comments can be helpful, in this scenario, the test is well written and formatted that it is easy to read and follow along. I would lean towards not including them here.